src/plonk_proof.rs

```diff
@@ -273,26 +274,26 @@ impl<C: HaloCurve, InnerC: HaloCurve<BaseField = C::ScalarField>> ProofTarget<C,
 }
 
 /// The opening of each Plonk polynomial at a particular point.
-#[derive(Clone, Debug)]
-pub struct OpeningSet<F: Field> {
```

Comment:
I've spent a good amount of time trying to make Serde work with `OpeningSet<F: Field>` without success. The compiler always barfs on weird trait bounds errors. All of our use cases always were of the form `OpeningSet<C::ScalarField>`, so hopefully changing to `OpeningSet<C: Curve>` won't make much of a difference.